modules/internal/ChapelIOStringifyHelper.chpl

```diff
@@ -22,14 +22,15 @@
    be called without IO and not cause circular dependencies.
 */
 module ChapelIOStringifyHelper {
-  private use ChapelStandard;
+  private use ChapelStandard, CTypes;
   private use BytesStringCommon only decodePolicy;
+  private use CTypes;
 
   @chpldoc.nodoc
   proc _can_stringify_direct(t) param : bool {
     if (t.type == string ||
         t.type == bytes ||
-        t.type == c_string ||
+        t.type == c_ptrConst(c_char) ||
```

Comment:
IMO this function should be using the raw `chpl_c_string`. I don't think it makes sense to treat `c_ptrConst(c_char)` specially here.

Reply:
without this, compilation of chapel programs fail, saying that:
```
modules/internal/ChapelLocale.chpl:744: error: use of 'rootLocale' before encountering its definition, type unknown
```

Reply:
Could you note changing it as a Future Work item in the PR message?

Reply:
noted this in the OP